src/main/java/com/github/manolo8/darkbot/gui/utils/window/WindowUtils.java

```diff
@@ -16,16 +19,21 @@ public static void setupUndecorated(JFrame mainFrame, JPanel mainPanel) {
         mainFrame.add(mainPanel);
 
         mainFrame.addComponentListener(new ComponentAdapter() {
+            Config.BotSettings.Window frame = ConfigEntity.INSTANCE.getConfig().BOT_SETTINGS.MAIN_GUI_WINDOW;
```

Comment:
this is NOT the place for this, at all.
This is a generic window util, that can make any window have no window decorations and be able to be resized. Both main and settings gui use this, and you should NOT make this tied to the config in any way